es-ds-styles/scss/_dropdown.scss

```diff
@@ -201,3 +201,137 @@
   color: variables.$dropdown-link-color;
 }
+
+// ==========================================
+// ES-DROPDOWN CUSTOM STYLES
+// ==========================================
+
+// Custom es-dropdown input styling
+.es-dropdown-input {
+  align-items: center;
+  background-color: variables.$input-bg;
+  background-clip: padding-box;
+  border: 1px solid variables.$gray-500;
+  border-radius: variables.$border-radius-xs;
+  color: variables.$input-color;
+  display: flex;
+  height: variables.$input-btn-height;
+  justify-content: space-between;
+  padding: 16px 16px 16px 32px;
+  font-family: variables.$input-font-family;
+  font-size: variables.$input-font-size;
+  font-weight: variables.$input-font-weight;
+  line-height: variables.$input-line-height;
+  text-decoration: none;
+  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  white-space: nowrap;
+  width: 325px;
+
+  &:hover, &:active {
+    border-color: variables.$blue-300;
+    .es-dropdown-panel {
+      border-top: 1px solid variables.$blue-300;
+    }
+  }
+
+  &:focus,
+  &:focus-visible {
+    border-color: variables.$gray-500;
+    outline: 0;
+    font-weight: variables.$font-weight-bold;
+  }
+
+  span[data-pc-section='input'] {
+    color: variables.$input-color-placeholder;
+    &:focus,
+    &:focus-visible {
+      outline: 0;
+      font-weight: variables.$font-weight-bold;
+    }
+  }
+
+  &:disabled,
+  &.disabled {
+    background-color: variables.$gray-50;
+    border-color: variables.$gray-500;
+    cursor: not-allowed;
+    span[data-pc-section='input'] {
+      color: variables.$gray-500;
+    }
+  }
+
+  &.es-dropdown-open {
+    border-bottom: none;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+}
+
+// Dropdown panel styling
+// Top corners have no radius so that it visually merges with the input
+.es-dropdown-panel {
+  background-color: variables.$white;
+  border: 1px solid variables.$gray-500;
+  border-radius: variables.$border-radius-xs;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  overflow: hidden;
+  // Force dropdown to always appear below with appendTo="self"
+  top: calc(100% - 57px) !important;
+  left: 15px !important;
+  width: 325px !important;
+}
+
+.es-dropdown-wrapper {
+  max-height: 300px;
+  overflow-y: auto;
+}
+
+.es-dropdown-list {
+  margin: 0;
+  padding: 0;
+}
+
+.es-dropdown-item {
+  align-items: center;
+  background-color: transparent;
+  border: 0;
+  color: variables.$gray-900;
+  cursor: pointer;
+  display: flex;
+  font-weight: variables.$font-weight-normal;
+  justify-content: space-between;
+  margin-bottom: 0;
+  padding: 16px 16px 16px 32px;
+  text-align: left;
+  transition: background-color 0.15s ease-in-out;
+  width: 100%;
+
+  &:hover {
+    background-color: variables.$blue-50;
+  }
+
+  &:active {
+    background-color: variables.$blue-100;
+    font-weight: variables.$font-weight-bold;
```

Comment:
If design agrees to change the focus state of the overall dropdown away from the bold and towards an outline, can you check with them to maybe revisit the bolding on "pressed" state as well? Because of the animation, the background color fades in more slowly than the font bolding (which takes effect immediately), meaning there are two pieces of the "pressed" state that arrive at different times, which looks odd to me. I might recommend to design that the background color change is enough.


https://github.com/user-attachments/assets/666f6406-b8b5-4541-98c4-6e55528876e1